src/components/MDXComponents/MDXCode.tsx

```diff
@@ -59,6 +59,9 @@ export const MDXCode = ({
   const shouldShowHeader = shouldShowCopy || title;
   const titleId = `${useId()}-titleID`;
   const codeId = `${useId()}-codeID`;
+  const defaultLineNumberValue = language !== 'bash'; //show line number by default for bash language
```

Comment:
Just thinking out loud, but maybe we can create a new variable that is an array of the languages we want to hide by default? Just in case we need to add more languages later?